### DISCUSSION TITLE
How does Lowdefy use Gantt charts？

### DISCUSSION THREAD

**Discussion by 15602322852** [lowdefy/lowdefy#1308] [Questions and Help]
How to use a Gantt chart?

**Comment by 13316090972** [comment]
I meet the same requirement.
We can find gantt component in react,e.g.,https://github.com/neuronetio/gantt-schedule-timeline-calendar,
and then convert it to lowdefy custom block,see https://docs.lowdefy.com/custom-blocks.
And finnally we can use the gantt charts in lowdefy.

**Reply by 15602322852** [reply]
Can the Amchart example be converted to Lowdefy？例如：https://www.amcharts.com/demos/gantt-chart-dates/#code

**Reply by 13316090972** [reply]
Unfortunately,it is hard to convert, for the example used several js functions.
Those chart which use pure json properties can be easilly converted to lowdefy block.for example,Echart.But Echart doesn't surport gantt,exactly time stack gantt.
![mmexport1664027481403](https://user-images.githubusercontent.com/56707831/192101754-f083c813-4a26-4564-8281-ef9b9d1aeef5.jpg)

**Reply by 15602322852** [reply]
ok,thank you!
